.scalafmt.conf

```
@@ -0,0 +1,41 @@
+# This file has been automatically generated and should
+# not be edited nor added to source control systems.
+
+# To edit the original configurations go to
+# https://github.com/alejandrohdezma/sbt-scalafmt-defaults/edit/master/.scalafmt.conf
+
+version = 3.0.5
+runner.dialect = scala213
+maxColumn = 120
+align.preset = none
+newlines {
+  sometimesBeforeColonInMethodReturnType = false
+}
+
+style = default
+
+assumeStandardLibraryStripMargin = true
+newlines.alwaysBeforeElseAfterCurlyIf = false
+binPack {
+  literalArgumentLists = true
+  literalsIncludeSimpleExpr = true
+  literalsExclude = [ "Term.Name" ]
+}
+
+docstrings.style = "SpaceAsterisk"
+docstrings.oneline = fold
+
+rewrite {
+  rules = [
+    PreferCurlyFors
+    RedundantBraces,
+    RedundantParens,
+    SortModifiers
+  ]
+  redundantBraces.maxLines = 1
+  redundantBraces.stringInterpolation = true
+}
+
+includeNoParensInSelectChains = false
+
+optIn.blankLineBeforeDocstring = true
```

```suggestion
optIn.blankLineBeforeDocstring = true

```

(newline at end of file)